### ISSUE TITLE
Applications identify Catppuccin-*-Dark as light theme

### ISSUE THREAD

**Issue by crillon** [catppuccin/gtk#100]
**Describe the bug**
Applications such as Google-chrome 114+, blackbox-terminal and some others identify Catppuccin-*-Dark as a light theme and use *light mode*.

**Mention your distribution and your theme version**
disro: Manjaro Xfce 23.0;
theme: 0.6.0-1;

**Select how you installed the theme**
- [ ] GitHub Releases
- [x] AUR package 
- [ ] Nixpkg
- [ ] Using the python install script 
- [ ] Flatpak

**Symlinked the gtk-4.0 folder?**
Yes.

**Expected behavior**
Catppuccin-*-Dark themes should be identified by applications as dark themes.

**Screenshots**
Chrome and blackbox when using the `Catpuccin-Mocha-Standart-Sky-Dark` theme:
![image](https://github.com/catppuccin/gtk/assets/43761130/db384826-64a7-46d8-ba93-e0998794b228)

To fix this, needs to make the names of all dark themes end with `-dark`
E.g. this is what applications look like when using the `Catpuccin-Mocha-Standart-Teal-dark` **(with lowercase 'd')** theme:
![image](https://github.com/catppuccin/gtk/assets/43761130/a4a64515-77d6-4e74-a7f8-e3eea901fbce)

**Comment by nekowinston** [comment]
https://github.com/catppuccin/gtk/commit/3fc72208deec4b7f8c22a2c77a21fde341e29cb6 seems to have caused other issues: https://github.com/catppuccin/gtk/issues/106#issuecomment-1636697137

**Comment by npv12** [comment]
> [3fc7220](https://github.com/catppuccin/gtk/commit/3fc72208deec4b7f8c22a2c77a21fde341e29cb6) seems to have caused other issues: [#106 (comment)](https://github.com/catppuccin/gtk/issues/106#issuecomment-1636697137)

Yes, though I would suggest keeping it non capitalised by default and let others capitalise it as necessary.
